client/src/components/CardPile/CardPile.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { FC } from 'react'
+import { CSSTransition, TransitionGroup } from 'react-transition-group'
+
+import CardComponent from '../Card'
```

Comment:
We should agree on a convention when we have 2 imports (component + type) with the same name. Do we:
- Name the component `CardComponent` and the type `Card`.
- Name the component `Card` and the type `CardType`.

Which one should be the default? I have no preferences for this.

Reply:
You could also rename your Card component to PlayingCard. One could confuse it with the standard card component you can find in libraries like Bootstrap.